core/math/basis.cpp

```diff
@@ -800,7 +800,7 @@ void Basis::set_quat(const Quat &p_quat) {
 void Basis::set_axis_angle(const Vector3 &p_axis, real_t p_phi) {
 // Rotation matrix from axis and angle, see https://en.wikipedia.org/wiki/Rotation_matrix#Rotation_matrix_from_axis_angle
 #ifdef MATH_CHECKS
-	ERR_FAIL_COND_MSG(!p_axis.is_normalized(), "Axis must be normalized.");
+	ERR_FAIL_COND_MSG(!p_axis.is_normalized(), "The axis Vector3 must be normalized by calling \".normalized()\" on the Vector3.");
```

Comment:
That doesn't work for C# users, so I think it's better not to mention anything language-specific. It's easy enough to figure out how to normalize a Vector3 or Quat.